packages/shared/src/components/auth/OnboardingSteps/ContentTypes/helpers.ts

```diff
@@ -0,0 +1,47 @@
+import { AdvancedSettings } from '../../../../graphql/feedSettings';
+import {
+  getContentCurationList,
+  getContentSourceList,
+  getVideoSetting,
+} from '../../../filters/helpers';
+import { Source } from '../../../../graphql/sources';
+
+interface GetContentTypeNotEmptyProps {
+  advancedSettings: AdvancedSettings[];
+  selectedSettings: Record<string, boolean>;
+  checkSourceBlocked: (source: Source) => boolean;
+}
+
+export const getContentTypeNotEmpty = ({
+  advancedSettings,
+  selectedSettings,
+  checkSourceBlocked,
+}: GetContentTypeNotEmptyProps): boolean => {
+  const contentSourceList = getContentSourceList(advancedSettings);
+  const contentCurationList = getContentCurationList(advancedSettings);
+
+  const videoSetting = getVideoSetting(advancedSettings);
+
+  const advancedSettingsSelected = (settings: AdvancedSettings[]) =>
+    !!settings
+      .map(({ id, defaultEnabledState }) => {
+        return selectedSettings[id] ?? defaultEnabledState;
+      })
+      .find((setting) => setting === true);
```

Comment:
I think we can simplify this and avoid the `!!` by using some.
```suggestion
    settings
      .map(({ id, defaultEnabledState }) => {
        return selectedSettings[id] ?? defaultEnabledState;
      })
      .some(Boolean);
```